build/template-run-device-auth-tests.yaml

```diff
@@ -11,9 +15,10 @@ steps:
   displayName: 'DeviceAuthTests'
   inputs:
     testSelector: 'testAssemblies'
-    testAssemblyVer2: '$(System.ArtifactsDirectory)\drop\**\**'
-    searchFolder: '$(System.ArtifactsDirectory)'
+    testAssemblyVer2: ${{ parameters.TestFilter }}
+    searchFolder: ${{ parameters.TestDirectory }}
     rerunFailedTests: true
     rerunMaxAttempts: '3'
     runInParallel: false
     diagnosticsEnabled: True
+    failOnMinTestsNotRun: true
```

Comment:
Where is the value for min test set? Is there some default?

Reply:
It defaults to 1 but it makes sense to specify it